app/Models/Ticket.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Ticket extends Model
+{
+    protected $fillable = ['title', 'content', 'slug', 'status', 'user_id'];
```

Comment:
```suggestion
    protected $fillable = [
        'title', 
        'content', 
        'slug', 
        'status', 
        'user_id'
    ];
```
mỗi tham số ở 1 dòng